test/Base.t.sol

```diff
@@ -58,19 +61,25 @@ abstract contract Base_Test is Assertions, Calculations, DeployOptimized, Modifi
 
     function setUp() public virtual {
         // Deploy the base test contracts.
+        contractWithoutReceiveEth = new ContractWithoutReceiveEth();
+        contractWithReceiveEth = new ContractWithReceiveEth();
         dai = new ERC20Mock("Dai Stablecoin", "DAI");
         noop = new Noop();
         recipientGood = new RecipientGood();
+        vm.deal({ account: address(recipientGood), newBalance: 100 ether });
```

Comment:
Why does `recipientGood` need ETH?

Reply:
Because it is used to call withdraw in some tests, and if it has a zero balance it would revert

Reply:
If it calls to `withdraw` then thats a reentrancy test. Don't we use `reentrantRecipient` for that? I thought `recipientGood` is the good recipient that does not try to reenter.